frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx

```diff
@@ -167,22 +168,30 @@ export const CreateStorageSystemFooter: React.FC<CreateStorageSystemFooterProps>
 
   const stepId = activeStep.id as number;
   const stepName = activeStep.name as string;
+  const { nodes } = state;
 
   const jumpToNextStep = canJumpToNextStep(stepName, state, t);
 
+  const handleStepId = () =>
+    dispatch({
+      type: 'wizard/setStepIdReached',
+      payload: state.stepIdReached <= stepId ? stepId + 1 : state.stepIdReached,
+    });
+
   const handleNext = async () => {
     const payloads = getPayloads(stepName, state, hasOCS, t);
-    if (payloads !== null) {
+    if (stepName === StepsName(t)[Steps.CreateLocalVolumeSet]) {
+      dispatch({
+        type: 'wizard/setCreateLocalVolumeSet',
+        payload: { field: 'showConfirmModal', value: true },
+      });
+    } else if (payloads !== null) {
       setRequestInProgress(true);
       try {
-        if (state.capacityAndNodes.nodes)
-          await Promise.all(labelNodes(state.capacityAndNodes.nodes));
+        if (nodes) await Promise.all(labelNodes(nodes));
         const requests = payloads.map(({ model, payload }) => k8sCreate(model as K8sKind, payload));
         await Promise.all([labelOCSNamespace(), ...requests]);
```

Comment:
Will this: https://github.com/openshift/console/pull/9409#discussion_r685835237 be handled in some other PR ?

Reply:
yeah in  PR #9630